packages/ERTP/src/displayInfo.js

```diff
@@ -61,7 +62,7 @@ export const coerceDisplayInfo = allegedDisplayInfo => {
     assert.equal(Reflect.ownKeys(allegedDisplayInfo).length, 0);
     assert.equal(Object.getPrototypeOf(allegedDisplayInfo), Object.prototype);
```

Comment:
If the `allegedDisplayInfo` that gets passed in was marked as `Data`, this prototype check will now fail (`Data` is now marked with a special inserted prototype, just like `Remotable`/`Far`).

I'd like someone who understands the intent better than me to think about whether that's the correct behavior or not.

To distinguish Data from Far, if it really matters, it might be best to use `passStyleOf()`.

Reply:
`allegedDisplayInfo` should ideally always be Data. In fact, all this code around lines 62-63 is trying to handle the degenerate case in which an empty object is wrongly treated as Far. If we think in this PR that we can prevent an empty object as being wrongly treated as Far, we can remove all of this code.

Reply:
I believe these lines can be changed to:

```js
export const assertDisplayInfo = allegedDisplayInfo => {
  const displayInfoKeys = harden(['decimalPlaces']);
  assertKeysAllowed(displayInfoKeys, allegedDisplayInfo);
};

export const coerceDisplayInfo = allegedDisplayInfo => {
  if (allegedDisplayInfo === undefined) {
    return undefined;
  }
  assert(passStyleOf(allegedDisplayInfo) === 'copyRecord');
  allegedDisplayInfo = pureCopy(allegedDisplayInfo);
  assertDisplayInfo(allegedDisplayInfo);
  return allegedDisplayInfo;
};
```

Reply:
@erights should we export a RECORD_STYLE (`= 'copyRecord'`) to match the `REMOTE_STYLE` we currently export? We'd need something stable if we want to be able to ask `passStyleOf` about pass-by-copy instead of asking about something not being pass-by-reference.

Reply:
@katelynsills sounds like a good change, I've filed #2553 to make that cleanup after we land this (and after we decide on a RECORD_STYLE export).

Reply:
See my response to @dtribble at https://github.com/Agoric/agoric-sdk/pull/2546#discussion_r583402307 :

> `REMOTE_STYLE` is an outlier because we know the name it has (`'presence'`) is not the name we want (`'remote'`). For the other's, the string literal style is pervasive in our code. We should discuss sometime the tradeoffs. There are a lot of places in JS where a misspelling can lead to something silently going astray, such as property or method dispatch. Our properjs typescript/jsdoc types help a lot with these. They also do OR typing of an enumerated set of string literals. So I don't know that the string-literal-as-enum is really a different hazard than property or method names. 
>
> We should definitely examine this question. But I'm not gonna take on a pervasive style change in this narrowly targeted PR.

My position remains that we should continue to use literal strings, and enlist the typing system to help us discover when we're doing it wrong. But I'm sure this is not settled and something we need to argue through, experiment with the typings of, and settle on one consensus style. Attn @dtribble @michaelfig 

In the meantime, please do not introduce a `RECORD_STYLE`.